crates/exex/exex/src/backfill.rs

```diff
@@ -195,38 +197,117 @@ where
     }
 }
 
+impl<E, DB, P> From<BackfillJob<E, DB, P>> for SingleBlockBackfillJob<E, DB, P> {
```

Comment:
i'm ok with it, but we still need an `into_single_blocks` method on `BackfillJob`